docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -478,6 +478,14 @@ The token bloom filter is the same as `ngrambf_v1`, but stores tokens (sequences
 tokenbf_v1(size_of_bloom_filter_in_bytes, number_of_hash_functions, random_seed)
 ```
 
+#### Sparse grams bloom filter {#sparse-grams-bloom-filter}
+
+The sparse grams bloom filter is the same as `ngrambf_v1`, but sparse grams tokens instead of ngrams.
```

Comment:
Could we link `sparse grams tokens` to https://clickhouse.com/docs/sql-reference/functions/string-functions#sparseGrams ?